inference-engine/src/mkldnn_plugin/mkldnn_plugin.cpp

```diff
@@ -296,6 +296,7 @@ static void Transformation(CNNNetwork& clonedNetwork, const Config& conf) {
 
     pass_config->enable<ngraph::pass::ConvertInterpolate1ToInterpolate4>();
     pass_config->enable<ngraph::pass::ConvertGather1ToGather7>();
```

Comment:
I think we need to update this transformation to have `ConvertGather1ToGather8` because once `Gather8` is supported on CPU you just remove triggering `ConvertGather8ToGather7`.